File: Source/net10 project/Test Files/VB/Inner Interface access.vb

' Test public class with an inner interface.
' The generator should include the public members of the inner interface in the summary.

#Disable Warning ' Disable all warnings in this file to test the generator.

Public Class TestClassPublicWithInnerInterface

    Public Interface TestInterfacePublic

        Sub TestMethodPublicNoModifier()

        Shadows Sub TestMethodShadows()

        Overloads Sub TestMethodOverloads()

    End Interface

End Class

#Enable Warning ' Re-enable all warnings.

File: Source/net10 project/Test Files/C#/Inner Interface access.cs

// Test public class with an inner interface.
// The generator should include the public members of the inner interface in the summary.

#pragma warning disable // Disable all warnings in this file to test the generator.

public class TestClassPublic
{
    public interface TestInterfacePublic
    {
        void TestMethodPublicNoModifier();

        new void TestMethodShadows();

        new void TestMethodOverloads();
    }
}

#pragma warning restore  // Re-enable all warnings.